Resources/config/propel.xml

```diff
@@ -35,30 +35,30 @@
             <tag name="data_collector" template="@Propel/Collector/propel" id="propel" />
         </service>
 
-        <service id="propel.twig.extension.syntax" class="%propel.twig.extension.syntax.class%">
+        <service id="propel.twig.extension.syntax" class="%propel.twig.extension.syntax.class%" public="true">
             <tag name="twig.extension" />
         </service>
 
-        <service id="form.type_guesser.propel" class="%form.type_guesser.propel.class%">
+        <service id="form.type_guesser.propel" class="%form.type_guesser.propel.class%" public="true">
             <tag name="form.type_guesser" />
         </service>
 
-        <service id="propel.form.type.model" class="%propel.form.type.model.class%">
+        <service id="propel.form.type.model" class="%propel.form.type.model.class%" public="true">
             <tag name="form.type" alias="model" />
         </service>
 
-        <service id="propel.dumper.yaml" class="%propel.dumper.yaml.class%">
+        <service id="propel.dumper.yaml" class="%propel.dumper.yaml.class%" public="true">
             <argument>%kernel.root_dir%</argument>
             <argument>%propel.configuration%</argument>
         </service>
 
-        <service id="propel.loader.yaml" class="%propel.loader.yaml.class%">
+        <service id="propel.loader.yaml" class="%propel.loader.yaml.class%" public="true">
             <argument>%kernel.root_dir%</argument>
             <argument>%propel.configuration%</argument>
-            <argument type="service" id="faker.generator" on-invalid="null" />
+            <argument type="service" id="faker.generator" on-invalid="null" public="true"/>
```

Comment:
I guess the `public="true"` is wrong here (`argument`s don't have a `public` attribute).